app/formats.js

```diff
@@ -0,0 +1,30 @@
+export default {
+  time: {
+    hhmmss: {
+      hour: "numeric",
+      minute: "numeric",
+      second: "numeric"
+    }
+  },
+  date: {
+    hhmmss: {
+      hour: "numeric",
+      minute: "numeric",
+      second: "numeric"
+    }
+  },
+  number: {
+    EUR: {
+      style: "currency",
+      currency: "EUR",
+      minimumFractionDigits: 2,
+      maximumFractionDigits: 2
+    },
+    USD: {
+      style: "currency",
+      currency: "USD",
+      minimumFractionDigits: 2,
+      maximumFractionDigits: 2
+    }
```

Comment:
Maybe also add CHF? Because if we use this, we will probably use it with CHF.

Reply:
Actually this is obsolete. `ember-intl` generated this but we won't use it for now..